### DISCUSSION TITLE
List of Prometheus Metrics generated by the collector

### DISCUSSION THREAD

**Discussion by karthikinamanamelluri** [open-telemetry/opentelemetry-collector#3726] [Q&A]
The collector is exposing prometheus metrics on port 8888 and endpoint /metrics. Is there any doc listing down the metrics that are being exposed. 

I was using 0.27.0 earlier and I see that 0.29.0 has different names for identifying namespace, pod name etc. I also notice that some metrics are missing 0.29.0.
Trying to find if there is any doc with information on this.

**Comment by nata7che** [comment]
joining to this request! documentation will be highly appreciated!
